gnomad_qc/v3/create_release/prepare_vcf_data_release.py

```diff
@@ -708,6 +714,18 @@ def prepare_vcf_header_dict(
         subset_list=subset_list,
         subset_pops=pops,
     )
+
+    # update VCF header for VRS fields
+    VRS_Allele_IDs_description = "The computed identifiers for the GA4GH VRS Alleles corresponding to the values in the alleles column"
+    VRS_Starts_description = "Interresidue coordinates used as the location starts for the GA4GH VRS Alleles corresponding to the values in the alleles column"
+    VRS_Ends_description = "Interresidue coordinates used as the location ends for the GA4GH VRS Alleles corresponding to the values in the alleles column"
+    VRS_States_description = "The literal sequence states used for the GA4GH VRS Alleles corresponding to the values in the alleles column"
```

Comment:
i know this is how it is stated in the google doc, but "columns" seems like the wrong term for a vcf ("fields" would be preferable)